stock_prices/stock_prices.py

```diff
@@ -1,15 +1,37 @@
 #!/usr/bin/python
 
 import argparse
+import math
+
+
+diff = []
+
 
 def find_max_profit(prices):
-  pass
+    global diff
+    diff = []
+
+    for i in prices:
+        find_diff(i, prices)
+
+    diff.sort()
+    return (diff[len(diff)-1])   
+       
+
+def find_diff(a, prices):
+    global diff
+    
+    for j in prices:
+        if prices.index(a) < prices.index(j):
+            diff.append(j-a)
+
```

Comment:
I love how you did this using global variables and a helper function. There might be another way to do this in one function by comparing minimum prices with maximum profit. But well done!